### PR TITLE
<StatisticsWidget/> - Add size prop

### PR DESCRIPTION
**Pull request by liorcode**
### 🔦 Summary

Added a "size" prop to statistics widget, with options (large,tiny) and default "large"
When size is "tiny", it would render a bold Text element

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX - Domas Markevičius
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [x] Cheatsheet (already exists)

- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

File: src/utils/AdaptiveHeading/AdaptiveHeading.js [hunk 1]
```diff
@@ -36,38 +37,40 @@ class AdaptiveHeading extends React.PureComponent {
       textInShort,
     } = this.props;
 
+    const Component = appearance === 'tiny' ? Text : Heading;
+
+    const sizingProps =
+      appearance === 'tiny'
+        ? { size: 'medium', weight: 'bold' }
+        : { appearance };
+
```
Comment (EgoziE) [inline src/utils/AdaptiveHeading/AdaptiveHeading.js:46]: Please extract a size-to-appearance map like before.

Component and sizingProps

File: src/StatisticsWidget/docs/index.story.js [hunk 2]
```diff
@@ -69,24 +70,6 @@ export default {
...
-
-          code({
-            compact: true,
-            autoRender: false,
-            title: 'descriptions',
-            description:
-              'Should be short, contains an explanation of the current stat.',
-            source: Descriptions,
-          }),
-
```
[the remark this answers is not in the document]
Comment (liorcode) [reply src/StatisticsWidget/docs/index.story.js:89]: oops 😟

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5791/

File: src/StatisticsWidget/StatisticsWidget.uni.driver.d.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { UniDriver } from 'wix-ui-test-utils/unidriver';
 
 export interface StatisticsWidgetUniDriver extends BaseUniDriver {
   getItemsCount: () => Promise<number>;
+  getSize: () => Promise<'large' | 'tiny'>;
```
Comment (EgoziE) [inline src/StatisticsWidget/StatisticsWidget.uni.driver.d.ts:6]: Size is a pure visual prop, there's no actual functionality that change here, so there's no need for a driver function for it.
For that we have visual tests (that I see you already added)

File: src/StatisticsWidget/StatisticsWidget.uni.driver.js [hunk 4]
```diff
@@ -41,6 +41,13 @@ const statisticsWidgetDriverFactory = (base, body) => {
     getItemsCount: async () =>
       await base.$$(getHookSelector(DataHooks.stat)).count(),
 
+    /** Get value size of the statistic */
+    getSize: async () => {
+      // index is irrelevant as all are the same, just take the first
+      const heading = await getAdaptiveHeadingDriver(0);
+      const tagName = await heading.getAppearance();
+      return tagName === 'SPAN' ? 'tiny' : 'large';
+    },
```
Comment (EgoziE) [inline src/StatisticsWidget/StatisticsWidget.uni.driver.js:50]: same comment

File: src/StatisticsWidget/test/StatisticsWidget.spec.js [hunk 5]
```diff
@@ -54,6 +54,13 @@ describe('StatisticsWidget', () => {
         expect(value).toBe('Changed value');
       });
 
+      it('should render tiny component', async () => {
+        const { driver } = render(<StatisticsWidget {...data} size="tiny" />);
+        const size = await driver.getSize();
+
+        expect(size).toBe('tiny');
+      });
+
```
Comment (EgoziE) [inline src/StatisticsWidget/test/StatisticsWidget.spec.js:63]: same comment

**Pull request merged**